src/index.js

```diff
@@ -6,19 +6,18 @@ const ganache = require('ganache-cli');
 const { deployContracts } = require('./contracts');
 
 program.on('--help', () => {
-  console.log();
-  console.log('  Examples:');
-  console.log();
-  console.log('    - Start a local Ethereum node on the default port');
-  console.log('      $ dav-cli --start');
-  console.log();
-  console.log('    - Start a local Ethereum node on port 1234');
-  console.log('      $ dav-cli --start --port 1234');
-  console.log();
-  console.log('    - Start a local Ethereum node on port 1234');
-  console.log('      $ dav-cli --port 1234');
-  console.log();
-  console.log('  Find out more at https://developers.dav.network');
+  const helpTex = `
```

Comment:
I'm a bit nit picking here... but the variable helpTex is weird, it wasn't intuitively clear to me that tex is short for text... I had to think about it... which kind of makes the shortening not worth it :)

I think we don't really need a variable here for now.

Can just:
```javascript
console.log(`
blah blah
`);
```

Reply:
Hey @TalAter Sorry for two commits. I did the github pr from the master branch and I accidentally pushed the other commit also to the same branch and it landed up here.  I will be careful next time. 
I agree we might not need a variable itself in this case. Will fix and commit. 

I couldn't find a way to remove the second commit. I am going to do the commit the changes here only. Please bear with me this one time 🙂